app/api.py

```diff
@@ -799,20 +799,23 @@ def get_element_text(self, doc, tag_name):
         else:
             return None
 
-    #Send hit to GA
-    def gamp(e,ec):
+    # ga_send_hit sends hit to Google Analytics via Measurment Protocol.
```

Comment:
Use doc string """...""" for method documents:
https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments